Misc/Positions_Area Positions 1 UK.txt

```diff
@@ -62,9 +62,24 @@ STC_M_CTR:London Control:199.998:SM:L:SCO:CTR:-:-::
 STC-M_CTR:London Control:199.998:SM:L:SCO-M:CTR:-:-::
 
 ;Military
-EGVV_CTR:Swanwick Military:135.150:VV:M:EGVV:CTR:-:-:6401:6457:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
-EGQQ_CTR:Scottish Military:134.300:QQ:M:EGQQ:CTR:-:-:4610:4667:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
-BLKDG_CTR:Blackdog:133.320:BKDG:M:BLKDG:CTR:-:-:6501:6577:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
-BLKDG_APP:Blackdog:133.320:BKDG:M:BLKDG:APP:-:-:6501:6577:N055.00.00.000:E001.00.00.000:N059.52.20.000:W001.16.27.000:N057.00.00.000:W003.00.00.000:N051.13.25.000:W003.49.06.000
+EGVV_CTR:Swanwick Mil:135.150:VV:M:EGVV:CTR:-:-:6401:6457:N059.52.20.000:W001.16.27.000:N056.30.20.000:W006.52.55.000:N053.24.11.000:W003.09.12.000
+
+;North
+EGVV_N_CTR:Swanwick Mil:134.300:VVN:M:EGVV:CTR:-:-:6401:6457:N057.01.39.274:W003.04.18.375::::
+EGVV_Q_CTR:Swanwick Mil:199.998:VVNW:M:EGVV:CTR:-:-:6401:6457:N057.01.39.274:W003.04.18.375::::
+
+;West
+EGVV_W_CTR:Swanwick Mil:133.900:VVW:M:EGVV:CTR:-:-:6401:6457:N051.10.45.296:W003.21.35.627::::
+EGVV_A_CTR:Swanwick Mil:199.998:VVSW:M:EGVV:CTR:-:-:6401:6457:N051.10.45.296:W003.21.35.627::::
+EGVV_C_CTR:Swanwick Mil:199.998:VVC:M:EGVV:CTR:-:-:6401:6457:N051.10.45.296:W003.21.35.627::::
+EGVV_D_CTR:Swanwick Mil:199.998:VVSE:M:EGVV:CTR:-:-:6401:6457:N051.10.45.296:W003.21.35.627::::
+
+;East
+EGVV_E_CTR:Swanwick Mil:133.320:VVE:M:EGVV:CTR:-:-:6401:6457:N053.56.05.631:E001.21.58.005::::
+EGVV_F_CTR:Swanwick Mil:199.998:VVNE:M:EGVV:CTR:-:-:6401:6457:N053.56.05.631:E001.21.58.005::::
```

Comment:
Unpublished/event/unlikely to be used splits are traditionally in the [Positions 3 UK](https://github.com/VATSIM-UK/UK-Sector-File/blob/master/Misc/Positions_Area%20Positions%203%20UK.txt) file

Using 199.998 will means that anyone logged on as EGVV_x_CTR _before_ they have set a primary frequency will have the ownership of VVNW and appear as online in the controller list. I would use 199.997/199.999

Reply:
We have reallocated the frequencies to the actual Swanwick ones.